### DISCUSSION TITLE
How to keep label always floating and visible placeholder?

### DISCUSSION THREAD

**Discussion by Gnuk** [material-components/material-web#5365] [Q&A]
Hi,

From 1.1.1, do you know how to keep label always floating with a visible placeholder on a `Text field`?

Ref: https://material-web.dev/components/text-field/.

Example of expectation before focus:

![a0839c17-eca4-470f-ad3a-c530f61a09a9](https://github.com/material-components/material-web/assets/1397032/baeabbdf-d586-43ca-b6a7-7ba3d33cab43)

_Note: From __Angular Material__, it's possible to have this behavior with `[floatLabel]="always"`._

Thank you for your help.

**Comment by asyncliz** [comment]
I filed https://github.com/material-components/material-web/issues/5368 for the feature, we don't have it yet :)
